SearchableTools/ContentView.swift

```diff
@@ -9,42 +9,25 @@ import SwiftUI
 
 struct ContentView: View {
     
-    @State var query = " "
-    let tools = [
-        "Martillo",
-        "Pala",
-        "Taladro",
-        "Destornillador",
-        "Alicate",
-        "Ratchet",
-        "Copa",
-        "Pulidora",
-        "Pinzas",
-        "Gato"
-    ]
+    @StateObject var viewModel = ViewModel()
+    @State private var query = " "
     
     var body: some View {
+        
         NavigationView {
+            
             List {
-                ForEach(filteredData, id: \.self) { tools in
-                    NavigationLink(destination: Text(tools)) {
-                        Text(tools)
+                
+                ForEach(viewModel.filteredData) {_ in
```

Comment:
El ForEach sigue igual, con los mismos componentes internos
```suggestion
                ForEach(viewModel.filteredData) { tool in
                  NavigationLink(destination: Text(tool)) {
                    Text(tool)
                    }
                }
```